aws-ecsfargate-terraform/README.md

```diff
@@ -229,6 +229,53 @@ resource "aws_ecs_task_definition" "op_scim_bridge" {
 
 If you need help with the configuration, [contact 1Password Support](https://support.1password.com/contact/).
 
+## Customize Redis
+
+As of SCIM Bridge `v2.8.5`, additional Redis configuration options are available. `OP_REDIS_URL` must be unset for any of these environment variables to be read. These environment variables may be especially helpful if you need support for URL-unfriendly characters in your Redis credentials. 
+
+> **Note**  
+> `OP_REDIS_URL` must be unset, otherwise the following environment variables will be ignored.
+
+* `OP_REDIS_HOST`: set the hostname of your redis cache (e.g., `"localhost"`)
+* `OP_REDIS_PORT`: Set the Redis port (e.g., `"6379"`)  
+* `OP_REDIS_USERNAME`: Set the username required to connect to Redis (e.g., `"admin"`)
+* `OP_REDIS_PASSWORD`: Set the password required to connect to Redis (e.g., `"apv.zbu8wva8gwd1EFC-fake.password"`). Can accommodate URL-unfriendly characters that `OP_REDIS_URL` may not accommodate. 
+* `OP_REDIS_SSL_ENABLED`: Set whether the connection to Redis should be TLS-enabled (Boolean `0` or `1`)  
+* `OP_REDIS_INSECURE_SSL`: Set whether insecure TLS connections to Redis should be permitted when `OP_REDIS_SSL_ENABLED` is set to `1`. (Boolean `0` or `1`)
```

Comment:
For this block, I might change `e.g.` to `default` (or explicitly state what the default values are) since default values _do_ exist, and it would also address @accraw's question about whether _all_ the broken-out variables are necessary or not (they aren't, due to those default variables).